README.md

```diff
@@ -28,46 +29,74 @@ Folder Structure:
-The tool adds a record to the `{current_datetime}.csv` file that will contain the file path to the invalid SQL and the error message returned by BigQuery. 
-E.G:
-`/invalid_sql/my_sql.sql, BigQuery Error Message.`
 
-If the dry-run succeeds, the tool copies the converted sql to the `validated_sql` folder specified in the config.py file.
+## Usage
+In order to utilize this tool, you first need to clone the project into the directory of your choice 
+`git clone https://github.com/RealistAI/bq_migration_automation_tool.git`, navigate into the newly cloned 
+directory `cd bq_migration_automation_tool`, and alter the config.py to your own specification. Create 
+a Pip virtual environment using `pipenv shell` and install the required libraries `pipenv install`, 
+and run the Makefile `make`
 
-## Git Integration
-If there are any new files in the `validated_sql` folder the tool does the following:
+## config.py
 
-Creates a new git branch in the UC4 SQL Repo named `bq_migration_tool_batch_{current_datetime}`.\
-Copies the SQL files from the `validated_sql` folder to the UC4 SQL Repo. \
-Commits changes and push those changes to Git.
+### DWH_MIGRATION_TOOL_REPO
+The repository url and branch containing the dwh-migration-tools
 
-## Usage
-To run the tool, the user will run the following command:\
-`git clone https://github.com/RealistAI/bq_migration_automation_tool.git` \
-`Make`
-
-The config file will contain default values for all the parameters we will need such as:\
-TRANSLATED_BUCKET\
-SQL_TO_VALIDATE\
-TARGET_SQL_PATH\
-CONFIG
+###UC4_SQL_REPO
+The repository url and branch containing the SQL's to translate
+
+### PROJECT = "michael-gilbert-dev"
+The name of the Google Cloud Platform project that will performing the bulk translation
+
+### PREPROCESED_BUCKET = "gs://dwh_preprocessed"
+A Google Cloud Storage bucket that will be used by `bqms-run` as a staging area for the translation process
+
+### SOURCE_SQL_PATH = Path(BASE_PATH, 'UC4_SQL', 'teradata_sql')
+The directory in your Github repository containing .sql files for translation and validation
+
+### TRANSLATED_BUCKET = "gs://dwh_translated"
+A Google Cloud Storage bucket that will be used by `bqms-run` to store translated files before dumping 
+them back into the local file system.
+
+### SQL_TO_VALIDATE = Path(os.getcwd(), 'transpiled_sql')
+The local directory that `bqms-run` will use to store the results of the run.
+
+## TARGET_SQL_PATH = Path(BASE_PATH, 'UC4_SQL', 'bigquery_sql')
+The directory within the origin Github repository to contain the translated and validated .sql files
+
+### CONFIG = Path(os.getcwd(), 'config', 'config.yaml')
+The path to the dwh-migration-tools config.yaml file. 
+
+### MAPPING_CONFIG_FILE = Path(os.getcwd(), 'config', 'name_mapping.json')
+The path to any given object name mapping configuration files.
+
+### FAILURE_LOGS = Path(os.getcwd(), 'failure_logs')
+The target destination for the failure logs contructed during the validation process.
```

Comment:
### UC4_SQL_REPO
The repository url and branch containing the SQL's to translate

### PROJECT
The name of the Google Cloud Platform project that will performing the bulk translation

### PREPROCESED_BUCKET
A Google Cloud Storage bucket that will be used by `bqms-run` as a staging area for the translation process

### SOURCE_SQL_PATH
The directory in your Github repository containing .sql files for translation and validation

### TRANSLATED_BUCKET
A Google Cloud Storage bucket that will be used by `bqms-run` to store translated files before dumping 
them back into the local file system.

### SQL_TO_VALIDATE
The local directory that `bqms-run` will use to store the results of the run.

## TARGET_SQL_PATH
The directory within the origin Github repository to contain the translated and validated .sql files

### CONFIG
The path to the dwh-migration-tools config.yaml file. 

### MAPPING_CONFIG_FILE
The path to any given object name mapping configuration files.

### FAILURE_LOGS
The target destination for the failure logs contructed during the validation process.